book/src/realworld.tex

```diff
@@ -446,153 +447,244 @@ \section{International Language Support}
+speaking UTF-8 natively, these problems have relaxed considerably.
 
-The package \pai{polyglossia}\cite{polyglossia} is a replacement for
-venerable \pai{babel} package. It takes care of the hyphenation patterns and automatically
-generated text strings in your documents.
-
-The package \pai{fontspec}\cite{fontspec} handles font loading for
-\hologo{XeLaTeX} and \hologo{LuaTeX}. The default font is Latin Modern
-Roman.
+The package \pai*{polyglossia} is a replacement for venerable \pai{babel}
+package. It takes care of the hyphenation patterns and automatically generated
+text strings in your documents. Polyglossia works only with \hologo{XeTeX}
+and \hologo{LuaTeX} engines.
 
 \subsection{Polyglossia Usage}
 
-Depending on the \TeX{} engine you use slightly different commands are
-necessary in the preamble of your document to properly enable multilingual
-processing. Figure~\ref{allinone} on page~\pageref{allinone} shows a sample preamble that takes care of all the necessary settings.
-
-\begin{figure}[!bp]
-\begin{lined}{10cm}
-\begin{verbatim}
-\usepackage{iftex}
-\ifXeTeX
-   \usepackage{fontspec}
-\else
-   \usepackage{luatextra}
-\fi
-\defaultfontfeatures{Ligatures=TeX}
-\usepackage{polyglossia}
-\end{verbatim}
-\end{lined}
-\caption[All in one preamble]{All in one preamble that takes care of \hologo{LuaLaTeX} and \hologo{XeLaTeX}} \label{allinone}
-\end{figure}
-
-So far there has been no advantage to using a Unicode \hologo{TeX} engine.
-This changes when we leave the Latin script and move to a more interesting
-language like Greek or Russian.  With a Unicode based system, you can
-simply\footnote{For small values of simple.} enter the native characters in your
-editor and \hologo{TeX} will understand them.
-
-Writing in different languages is easy, just specify the languages in the
-preamble. This example uses the \pai{csquotes} package which generates
-the right kind of quotes according to the language you are writing in. Note
-that it needs to be loaded \emph{before} loading the language support.
+So far there has been no advantage to using a Unicode \hologo{TeX} engine. This
+changes when we leave English and move to a language that uses non-Latin
+alphabet like Greek or Russian. With a Unicode based system, you can
+simply\footnote{For small values of simple.} enter the native characters in
+your editor and \hologo{TeX} will understand them.
 
+Writing in different languages is easy, just load the \pai{polyglossia} package
+and specify the languages in the preamble using
 \begin{lscommand}
-\verb|\usepackage[autostyle=true]{csquotes}|\\
-\verb|\setdefaultlanguage{english}|\\
-\verb|\setotherlanguage{german}|
+  \ci{setdefaultlanguage}[options][language] \\
+  \ci{setotherlanguage}[options][language]
 \end{lscommand}
-%
-To write a paragraph in German, you can use the German environment:
+where language is either the name of the language to use such as \cargv{gaelic}
```

Comment:
```suggestion
where \carg{language} is either the name of the language to use such as \cargv{gaelic}
```

not sure if this is correct ... but since language is an argument I would expect it to use the same formatting as in the lscommand definition